pkgs/games/steam/chrootenv.nix

```diff
@@ -103,7 +106,15 @@ in buildFHSUserEnv rec {
     export TZDIR=/etc/zoneinfo
   '';
 
-  runScript = "steam";
+  runScript = writeScript "steam-wrapper.sh" ''
+    #!${stdenv.shell}
+    glxinfo >/dev/null 2>&1
+    if [ ! "$?" = "0" ]; then
+      echo "*** WARNING: Test for 32-bit libGL unsuccessful."
+      echo "             This could mean you forgot to activate hardware.opengl.driSupport32Bit"
```

Comment:
This should go into `stderr`. Also,  heredoc would be preferrable:

```
cat <<EOF > /dev/stderr
WARNING: Test for 32-bit libGL unsuccessful.
This could mean you forgot to activate hardware.opengl.driSupport32Bit
EOF
```